### PR TITLE
Fix Symfony 6 Exception

### LINKED ISSUE #506: Submitting form using GET method with non-scalar parameters
**Sylius version affected**: 1.10.0

**Description**  
Due to Symfony deprecation mentioned in  #335:

`
User Deprecated: Since symfony/http-foundation 5.1: Retrieving a non-string value from "Symfony\Component\HttpFoundation\InputBag::get()" is deprecated, and will throw a "Symfony\Component\HttpFoundation\Exception\BadRequestException" exception in Symfony 6.0, use "Symfony\Component\HttpFoundation\InputBag::all($key)" instead.
`

using Symfony Form (with GET method) with non-scalar parameters will throw an Exception in Symfony 6.
Form example: https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/blob/master/src/Form/Type/SearchType.php

When I try to submit this form:
Screenshot 2022-11-28 at 20 36 10

 In Symfony 5.4 same form works perfectly.

**Steps to reproduce**  
Try to submit Form from the example above or create any form with array parameters using GET method.

**Possible Solution**  
https://github.com/Sylius/SyliusResourceBundle/blob/4c185126e1783ff9a85c9195d0c3466311f01574/src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php#L63

should be:
```php
$data = $request->query->all()[$name];
```

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #506
| License         | MIT

### CODE REVIEW

File: src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ public function handleRequest(FormInterface $form, mixed $request = null): void
                     return;
                 }
 
-                $data = $request->query->get($name);
+                $data = $request->query->all()[$name];
```
Comment (jakubtobiasz) [inline src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php]:
```suggestion
                $data = $request->query->all($name);
```

Cosmetic change 😅

**Review by jakubtobiasz — APPROVED** [review]

File: src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php (hunk 1 above)
Comment (lchrusciel) [reply src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php]: @mpysiak note for the future, please either apply suggestions from the comment or answer why you don't want to do so. Please, don't just mark a comment as resolved without anything in it

**Review by lchrusciel — APPROVED** [review]

File: src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php (hunk 1 above)
Comment (mpysiak) [reply src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php]: @lchrusciel sorry, it was my first PR 😉

**Comment by lchrusciel** [comment]
Hey MIchał, thank you very much for the PR! 🎉  And welcome among other Sylius contributors! :)

File: src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php (hunk 1 above)
Comment (csabavirag) [reply src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php]: @lchrusciel, it looks like this "cosmetic change" breaks in SF 5.4. Although it works in SF6. It produces an error
`Unexpected value for parameter "xxx": expecting "array", got "string"`

It works fine if I revert back to 
```php
$data = $request->query->all()[$name];
```

**Pull request merged**